test/test_settings.py

```diff
@@ -350,3 +350,120 @@ def test_cannot_set_invalid_values_for_max_header_list_size(self, val):
 
             with pytest.raises(KeyError):
                 s[h2.settings.MAX_HEADER_LIST_SIZE]
+
+class EqualityTestsMixin(object):
+    """
+    A mixin defining tests for the standard implementation of C{==} and C{!=}.
+    """
+
+    def anInstance(self):
+        """
+        Return an instance of the class under test.  Each call to this method
+        must return a different object.  All objects returned must be equal to
+        each other.
+        """
+        return h2.settings.Settings()
+        
+
+    def anotherInstance(self):
+        """
+        Return an instance of the class under test.  Each call to this method
+        must return a different object.  The objects must not be equal to the
+        objects returned by C{anInstance}.  They may or may not be equal to
+        each other (they will not be compared against each other).
+        """
+        return h2.settings.Settings()
```

Comment:
So this method must return a meaningfully different object than the one above it. The easiest way to do that is to use the `client` and `initial_values` arguments to the [`Settings` constructor](https://github.com/python-hyper/hyper-h2/blob/master/h2/settings.py#L100) to create a meaningfully different object.